packages/fxa-auth-server/test/local/payments/configuration/manager.js

```diff
@@ -113,6 +113,7 @@ describe('PaymentConfigManager', () => {
       productConfigId: testProductId,
     };
     await planConfigDbRef.doc(testPlanId).set(testPlanConfig);
+    await paymentConfigManager.maybeLoad();
```

Comment:
Sometimes the merged config loader would fail because the config had not been loaded, this happens depending on system speed because the constructor throws out a promise it can't wait for (constructors cannot be async) to load it. Waiting for `maybeLoad` here reduces flakey behavior by ensuring the configuration is loaded.